src/app/Library/CrudPanel/Traits/Update.php

```diff
@@ -52,17 +52,7 @@ public function getUpdateFields($id = false)
         $entry = ($id != false) ? $this->getEntry($id) : $this->getCurrentEntry();
 
         foreach ($fields as &$field) {
-            // set the value
-            if (! isset($field['value'])) {
-                if (isset($field['subfields'])) {
-                    $field['value'] = [];
-                    foreach ($field['subfields'] as $subfield) {
-                        $field['value'][] = $entry->{$subfield['name']};
-                    }
```

Comment:
All of this was for nothing, since there was a bit of code treating the `date_range` field below, doing the exact same thing if the field name is an array.